spec/dummy/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  include Wor::Paginate
+
+  def index
+    render_paginated Post, preserve_records: true
+  end
+
+  def index_by_id
+    render_paginated Post, preserve_records: { by: :id }
```

Comment:
:rocket: :smile: :tada: Nice!! Can we add these examples and many docs in the readme?